### DISCUSSION TITLE
publisher not working after setting writer qos ownership to exclusive.

### DISCUSSION THREAD

**Discussion by mahesh2392** [OpenDDS/OpenDDS#3976] [General]
Hi I am new to dds(3.22), I am have two writers in different applications running on different machines I am trying to set qos ownership of writer i.e wr1 and wr2 to exclusive so based on strengths of wr1 or wr2 reader which resides 3rd application will receive data from either wr1 or wr2(At a time single writer can write data those having strength more than other) but once i set ownership qos to writer it won’t publish anything once i removed this qos reader can receive data from both writer for same topic….what is the issue here?? Thanks

**Comment by mitza-oci** [comment]
There is an Ownership QoS demo here https://opendds.org/quickstart/shapes/ownership.html please check if the way you're using the middleware is significantly different.

**Comment by mahesh2392** [comment]
> There is an Ownership QoS demo here https://opendds.org/quickstart/shapes/ownership.html please check if the way you're using the middleware is significantly different.

It is working once i set same ownership qos to reader but once I stop wr1 with high strength it wont  connect to wr2…

**Comment by jrw972** [comment]
Please create a PR where your scenario is added as a test.  Please see https://github.com/objectcomputing/OpenDDS#how-to-make-a-minimum-working-example
